db/migrate/20220121104214_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :authorid
```

Comment:
- [ ] Following the ERD. This attribute is not required. It will be nice to remove it. Instead, it will be added as a foreign_key.

Reply:
This issue is well resolved! ✔️